dev-client/src/screens/ProjectListScreen/components/ProjectList.tsx

```diff
@@ -60,4 +60,4 @@ export const ProjectList = () => {
 };
 
 const keyExtractor = (project: Project) => project.id;
-const ItemSeparatorComponent = () => <Box h={`${SEPARATOR_HEIGHT}px`} />;
+const ItemSeparatorComponent = () => <Box height="sm" />;
```

Comment:
question: what does small mean for box size?

Reply:
see here in the theme file: https://github.com/techmatters/terraso-mobile-client/pull/1133/files#diff-49f32694a25aa6dfec4430b928176069a352f81c813aab5155de14cfa6919f47R82-R88